composables/event.ts

```diff
@@ -19,11 +19,15 @@ export function useEvent(id: MaybeRef<string>) {
   const firebaseCurrentUser = useCurrentUser()
   const queryClient = useQueryClient()
   return useQuery({
-    queryKey: queryKeyFactory.events,
+    queryKey: queryKeyFactory.event(toValue(id)),
     queryFn: () => $api<EventWithAttendees>(`/api/event/${toValue(id)}`),
     enabled: computed(() => !!firebaseCurrentUser.value), // Only run when user exists
     placeholderData() {
       return queryClient.getQueryData<EventWithAttendees[]>(queryKeyFactory.events)?.find(event => event.id === id)
     },
   })
 }
+
+export function typeIsEventWithAttendees(obj: EventWithAttendees | unknown): obj is EventWithAttendees {
```

Comment:
What is this used for?

Reply:
Might have been used once, but forgot to remove